### PR TITLE
Add update listin event

### PR DESCRIPTION
**Pull request by johnykes**

### CODE REVIEW

File: src/modules/rabbitmq/blockchain-events/handlers/updatePrice-event.handler.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ import { MarketplaceTypeEnum } from 'src/modules/marketplaces/models/Marketplace
 import { UsdPriceService } from 'src/modules/usdPrice/usd-price.service';
 import { BigNumberUtils } from 'src/utils/bigNumber-utils';
 import { DEADRARE_KEY } from 'src/utils/constants';
```
Comment (danielailie) [inline src/modules/rabbitmq/blockchain-events/handlers/updatePrice-event.handler.ts:15]: here you need to revert the changes. update price event handler should remain the same

**Review by danielailie — CHANGES_REQUESTED** [review]

File: src/modules/rabbitmq/blockchain-events/handlers/updatePrice-event.handler.ts (hunk 1 above)
Comment (johnykes) [reply src/modules/rabbitmq/blockchain-events/handlers/updatePrice-event.handler.ts]: Done

**Review by danielailie — APPROVED** [review]

**Review by gabrielmatei — APPROVED** [review]
